subjects/string_literals/README.md

```diff
@@ -0,0 +1,53 @@
+## string literal
```

Comment:
for the `string_literal` exercise, the name here was correct the wrong naming was done on the migration. I will avoid the migration and change the name in the rest of the files

Reply:
@Frenchris is this pull request finished? I would like to merge this pull request before you make new ones. I don't want it to get messy with a bunch of pull requests open at the same time.